draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
The client hasn't created any streams and then sends:

P10: [Stream=1, UNI=1]
P11: [Stream=3, UNI=Whatever]

But P10 gets lost, so the server just sees P11, and now it knows that Stream 1 is open, but it doesn't know what its state is. in QUIC currently, it can start sending, but now it can't because if it does, it creates an error when the client gets the server's data.